includes/image/Image.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+interface WPNotify_Image extends JsonSerializable {
```

Comment:
This class also validates the `WPNotify_JsonUnserializable` interface.

Reply:
fixed